.github/workflows/tests.yml

```diff
@@ -133,43 +133,41 @@ jobs:
             _build/default/js/engine/*.bc.js
             _build/default/js/languages/*/*.bc.js
 
-  #TODO: commented for now because prevent the release and
-  # those jobs get "cancelled" in each of our PR.
-  #  test-semgrep-js:
-  #    name: test semgrep js
-  #    needs: [build-semgrep-js-ocaml-test]
-  #    runs-on: ubuntu-22.04
-  #    container: emscripten/emsdk:3.1.46
-  #    env:
-  #      HOME: /root
-  #    steps:
-  #      - name: Restore git checkout cache
-  #        id: restore-git
-  #        uses: actions/cache/restore@v3
-  #        with:
-  #          path: .
-  #          key: semgrep-with-submodules-and-tree-sitter-${{ github.sha }}
-  #      - name: Make checkout speedy
-  #        if: ${{ steps.restore-git.outputs.cache-hit != 'true' }}
-  #        run: git config --global fetch.parallel 50
-  #      - uses: actions/checkout@v3
-  #        if: ${{ steps.restore-git.outputs.cache-hit != 'true' }}
-  #        with:
-  #          submodules: true
-  #      - name: Set up tree-sitter
-  #        if: ${{ steps.restore-git.outputs.cache-hit != 'true' }}
-  #        run: |
-  #          (cd libs/ocaml-tree-sitter-core && ./configure && ./scripts/install-tree-sitter-lib)
-  #      - uses: actions/download-artifact@v3
-  #        with:
-  #          name: semgrep-js-ocaml-test-${{ github.sha }}
-  #          path: _build/default/js
-  #      - uses: actions/setup-node@v3
-  #        with:
-  #          node-version: "18"
-  #      - name: Run semgrep js e2e tests
-  #        run: |
-  #          make -C js -j $(nproc) test
+  test-semgrep-js:
+    name: test semgrep js
+    needs: [build-semgrep-js-ocaml-test]
+    runs-on: ubuntu-22.04
+    container: emscripten/emsdk:3.1.46
+    env:
+      HOME: /root
+    steps:
+      - name: Restore git checkout cache
+        id: restore-git
+        uses: actions/cache/restore@v3
```

Comment:
I think this action only does the restore, but not the save. Is the cache being saved elsewhere? If not, I think the `actions/cache@v3` action will restore here and also trigger a save if the job completes successfully.

Reply:
Cache is being saved in previous step

Reply:
Wait no

Reply:
Wait yes

Reply:
Ohh got it. I thought this was a cache just for this step, but I recall having a conversation about this last week.